data/categories.txt

```diff
@@ -29,6 +29,7 @@ category "ship"
 	"Unclassified"
 	"Unclassified Minor"
 	"Game Mechanics"
+	"Intrasolar"
```

Comment:
Is this category still needed as it's not used?

Reply:
If the code for both intrasolar category and attribute are merged, I think it'd be better to have the category already defined so it can be documented (with comment in categories.txt and/or wiki) and in-place for usage rather than just something hidden in the code, even if there are presently no ships or variants defined.

Reply:
Yes, having the category defined in anticipation of having ships to go in it is OK.